SB2018/2018-02-07.md

```diff
@@ -0,0 +1,35 @@
+
+# Kit updates
+This release contains many improvements to the kit, and what you can do with it. The most notable changes are:
+- Reduce the verbosity of logs
+- New coordinate systems
+- Support for [custom arduino commands](https://docs.sourcebots.co.uk/api/servo-assembly/arduino-commands/)
+
+A more detailed changelog for this release is available from [the docs](https://docs.sourcebots.co.uk/updates/jan-2018/).
+
+To update your kit, you can either use the update file or flash the SD card. We highly recommend using the update file. Instructions on [what to do with our update file](https://docs.sourcebots.co.uk/kit/pi/update-file/), [how to flash an SD card](https://docs.sourcebots.co.uk/kit/pi/sd-card/), and the related [download links](https://docs.sourcebots.co.uk/updates/jan-2018/), are available from [the docs](https://docs.sourcebots.co.uk/kit/pi/#updating-your-pi). if you have any issues updating, be sure to consult the documentation, or [contact us](techsupport@sourcebots.co.uk).
+
+## The new coordinate systems
+In this release, we've introduced 2 new coordinate systems, and depricated an existing one.
+
+As many have noted the [`cartesian`](https://docs.sourcebots.co.uk/api/vision/coordinates/#cartesian-coordinates) system was not implemented, and would raise an exception whenever accessed. This has now been fixed.
+ 
+The `polar` coordinate system has been replaced with [`spherical`](https://docs.sourcebots.co.uk/api/vision/coordinates/#spherical-coordinates). The new system is easier to understand and better documented than polar. We recommend updating as soon as possible! Be sure to consult the documentation on exactly now to use the new system. Attempting to use `polar` will still work as it does now, however prints a warning message.
+ 
+# Vision accuracy
+We've had many teams contact us, saying the current vision implementation is unaccurate. This is made worse by the innacuracies in the `polar` system mentioned above, however is not the primary factor. We are actively looking into improving the accuracy, and hope to update you on this in the next few weeks.
+
+# The Documentation
+Our [documentation](https://docs.sourcebots.co.uk) is a great resource to learn how to use our kit, both the hardware and software components. It's highly detailed, and as a result, may contain errors. If you find an issue in the API, think you can word something better, or find any other issue, be sure to let us know. For those keen enough, feel free to [raise an issue directly](https://github.com/sourcebots/docs/issues), or [submit a pull request](https://github.com/sourcebots/docs/pulls).
+
+# Code restarting tip
+At our Tech Day last weekend, we noticed teams restarting their power board before making changes, this is unnecessary. The kit has been designed so that simply reinserting the USB drive will trigger the code to restart. 
```

Comment:
Nitpick, but it might be nice to make it clear that reinserting the USB drive is much faster than rebooting the power board